src/buffer/channel.rs

```diff
@@ -35,11 +35,17 @@ pub fn view<'a>(
             |message| match &message.source {
                 data::message::Source::Channel(_, kind) => match kind {
                     data::message::ChannelSender::User(user) => {
+                        let datetime = selectable_text(
+                            &message
+                                .timestamp
+                                .with_timezone(&chrono::Local)
+                                .format("[%H:%M] "),
```

Comment:
should we make a helper method so we save `DateTime<Utc>` but return `DateTime<Local>`?